ajout tuto

Vorpalin opened the pull request:
ajout tuto (explication du jeu à l'intérieur)
réglage de bug
ajout graphique

Vorpalin committed:
ajout

Vorpalin committed:
reajout multi

Vorpalin committed:
réintroduction multi

Vorpalin committed:
ajuot_complement

Vorpalin committed:
amelioration_et_bug_fixed

clementfrg commented:
@Vorpalin Est ce que tu continue cette branch ou tu as fini ce que tu voulais y faire ?

clementfrg commented on .gitignore:
```diff
@@ -25,8 +25,6 @@
 
 # Visual Studio cache directory
 .vs/
-.idea/
```
Pourquoi faire ??

clementfrg replied:
Je comprend pas pouruqoi tu veux les avoirs, ce sont des trash files

clementfrg commented on Assets/Mirror/Authenticators/BasicAuthenticator.cs:
C'est le retours de miror ? Il y as eu des changements ?

clementfrg commented:
Une fois ces réponses, je merge :)

clementfrg reviewed (commented):
Je merge une fois fonctionnel pour tous

Vorpalin committed:
Update gitignore

ultime27 commented:
problème de limite de map pour le perso (il y a bien des limites mais on peut quand meme les passer) et on voit le logo:

<img width="1009" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/0e7ebd3f-2d31-4443-b5d4-1199231f39e8">

problème interaction avec la caisse (on peut rien faire):

<img width="1009" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/56ea3b01-bf28-4031-ada1-0a438e5d249b">

bug quand on achète la quantité n'augmente pas:

<img width="1273" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/14d5e22b-075a-482b-a0cd-baaa22110a55">

bug a l'achat des cartes de fidélité:

<img width="1273" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/e5ac65ee-b610-4e46-a7a2-ba6dae308fcb">

bug quand on achète champoing+coloration:

<img width="1273" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/2eda9a47-428f-4ae2-80b5-477f850629f1">

erreurs tout le temps la :

<img width="1273" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/8adb949e-728f-4b7b-9103-1c924196749a">

le tour ne change pas a cause de pb d'attractivité:(erreur en boucle):

<img width="1273" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/4750009a-4fd1-4284-ac20-4a8b7d286519">

changement taille paramètre serveur en fct des réglages de qualité:
360p:
<img width="1497" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/a61e9bdf-5493-4a54-809c-b45fcb79fb49">
1080p:
<img width="1497" alt="image" src="https://github.com/SimuCorp/SIMUCORP/assets/45576461/7f1d6a82-088e-4a14-bd27-6382fc54417f">

Vorpalin committed:
bug_fixed

clementfrg commented:
@ultime27 Tu testeras le soucis avec la scène et redimensionnement

Vorpalin committed:
changemgement

Vorpalin committed:
ajout_lobby_et_crédit

ajout animation;
ajout lobby (plus besoin celui de mirror donc commencer sur la scène lobby)
ajout crédit (éteint l'application à son terme)
bug fixed

Vorpalin committed:
ajout_cinméatique et bug_fixed

clementfrg commented on .gitignore:
```diff
@@ -26,7 +26,6 @@
 # Visual Studio cache directory
 .vs/
 .idea/
-.vsconfig
```
Pourquoi ????

clementfrg reviewed (changes requested):
Juste le gitignore encore

Vorpalin committed:
Update gitignore

Vorpalin committed:
ajout_malus et bug_fixed

malus : ajouter par le biais des actions commerciales

Vorpalin committed:
simplification_tuto et bug_fixed

Vorpalin committed:
redimensionnement argent et bug fixed

Vorpalin committed:
bug_fixed

Vorpalin committed:
fix typo
